src/backends/ilp-quoter/index.js

```diff
@@ -30,9 +30,13 @@ class ILPQuoter {
 
   * putPair (uri) {
     const req = request({ method: 'PUT', uri, json: true })
-    const res = yield req
-    if (res.statusCode >= 400) {
-      throw new UnsupportedPairError(req.uri)
+    try {
+      const res = yield req
+      if (res.statusCode >= 400) {
```

Comment:
We should treat 4xx and 5xx differently as Alan suggests, otherwise you will get very confusing errors when the server is down